models/fivetran_log__mar_table_history.sql

```diff
@@ -57,7 +54,6 @@ latest_mar as (
         (free_monthly_active_rows + paid_monthly_active_rows) as total_monthly_active_rows
```

Comment:
For this release I would recommend we keep it to `measured_at` otherwise this will be a breaking change and we should get the update out in a patch release.

This is something we can open an FR for to clarify in the next breaking change.